nailgun/entities.py

```diff
@@ -2361,6 +2363,25 @@ def create_payload(self):
             u'operatingsystem': super(OperatingSystem, self).create_payload()
         }
 
+    def update(self, fields=None):
```

Comment:
This method is unnecessary.

```python
>>> from nailgun import entities
>>> os = entities.OperatingSystem(id=10).read()
>>> os.name
'127548037869023'
>>> os.name = '474523625643091'
>>> os = os.update(['name'])
>>> os.name
'474523625643091'
```

When we submit a PUT request to Satellite, we expect Satellite to respond with a full description of the updated entity. This method is only necessary if Satellite fails to return a full description.